cli/flox-rust-sdk/src/models/manifest.rs

```diff
@@ -1032,6 +1035,73 @@ pub enum ManifestBuildSandbox {
     Pure,
 }
 
+#[derive(Debug, Clone, Serialize, Deserialize, PartialEq, Default)]
+#[serde(deny_unknown_fields)]
+#[cfg_attr(test, derive(proptest_derive::Arbitrary))]
+pub struct ManifestContainerize {
+    #[serde(skip_serializing_if = "Option::is_none")]
+    pub config: Option<ManifestContainerizeConfig>,
+}
+
+/// Container config derived from
+/// https://github.com/opencontainers/image-spec/blob/main/config.md
+///
+/// Env and Entrypoint are left out since they interfere with our activation implementation
+/// Deprecated and reserved keys are also left out
+#[derive(Debug, Clone, Serialize, Deserialize, PartialEq, Default)]
+#[serde(rename_all = "PascalCase")]
+#[serde(deny_unknown_fields)]
+#[cfg_attr(test, derive(proptest_derive::Arbitrary))]
+pub struct ManifestContainerizeConfig {
```

Comment:
I took a look at using https://docs.rs/oci-spec/0.7.1/oci_spec/image/struct.Config.html but it doesn't derive proptest and between that and not wanting Env and Entrypoint it didn't seem worth using